llm/mcp_servers/services/resume_loader_service.py

```
@@ -0,0 +1,70 @@
+        return file_text
+
+    async def get_resume(self):
+        resume_path, applicant_name = self.find_resume_file()
+        if resume_path is None:
+            logging.error(f"No .txt resume file found in {self.RESUME_FILES_PATH_PREFIX}")
+            return None, None
+        resume = await self.read_file(resume_path)
+        return resume, applicant_name
+    
+    def find_resume_file(self):
+        for root, dirs, files in os.walk(self.RESUME_FILES_PATH_PREFIX):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)